crates/context_aware_config/src/api/default_config/handlers.rs

```diff
@@ -15,6 +15,7 @@ use crate::{
         context::helpers::validate_value_with_function,
         default_config::types::DefaultConfigKey,
         functions::helpers::get_published_function_code,
+        type_templates::types::QueryFilters,
```

Comment:
why we are using type_templates queryfilter , can we make this generic listquery type